homeassistant/components/homematicip_cloud/generic_entity.py

```diff
@@ -194,7 +211,13 @@ def available(self) -> bool:
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return f"{self.__class__.__name__}_{self._device.id}"
+        unique_id = f"{self.__class__.__name__}_{self._device.id}"
+        if self._channel:
+            unique_id = (
+                f"{self.__class__.__name__}_Channel{self._channel}_{self._device.id}"
+            )
+
+        return unique_id
```

Comment:
Does unique_id stay the same for existing entities?

Reply:
Good point. I added a small fix and now the unique_id does not change.